test/spreadProp.js

```diff
@@ -0,0 +1,34 @@
+import RA from '../src/index';
+import eq from './shared/eq';
+
+
+describe('spreadProp', function() {
+  it('tests currying', function () {
+    const prop = 'b';
```

Comment:
line 7 to 14 are not needed here IMHO. We are testing currying here only. The body of this test could be following
```javascript
it('tests curring', function () {
  eq(RA.spreadProp('prop', {}), {}); 
  eq(RA.spreadProp('prop')({}), {});
});